### DISCUSSION TITLE
微软key部署问题

### DISCUSSION THREAD

**Discussion by qzdemon** [lobehub/lobehub#2800] [Development Question]
再部署lobechat的docker服务的时候使用的是微软的key，现在我需要加入gpt4-o的模型 是需要重新部署一个docker服务吗？
还有就是在部署一个docker服务的时候能不能使用两个微软的key  因为4-o模型是另一个key

**Comment by arvinxx** [comment]
目前不支持，建议考虑接入类似one-api这样的服务

**Reply by qzdemon** [reply]
好的！ 谢谢
